### DISCUSSION TITLE
Feature branch during execution

### DISCUSSION THREAD

**Discussion by DerpSnepbek** [DerpSnepbek/Tafe24S1T112_Ext#3] [General]
Hey Sunny,

Not entirely sure what the assignment means by 'screenshot of the feature branch during execution'.
![image](https://github.com/DerpSnepbek/Tafe24S1T112_Ext/assets/161171900/4de45b9e-276d-48a7-812a-6e7cecdea825)

Do you think that just means 'show the program running'?
Just wondering if you took a screenshot for this one, and what you used?

Thanks

**Comment by Sunnyyu33a** [comment]
That's what I did, I think so, just with some test values in and execute
